tsdb/engine/tsm1/engine.go

```diff
@@ -1843,7 +1843,7 @@ func (e *Engine) createVarRefSeriesIterator(ref *influxql.VarRef, name string, s
 		for i, ref := range opt.Aux {
 			// Create cursor from field if a tag wasn't requested.
 			if ref.Type != influxql.Tag {
-				cur := e.buildCursor(name, seriesKey, &ref, opt)
+				cur := e.buildCursor(name, seriesKey, tfs, &ref, opt)
```

Comment:
We should definitely avoid reparsing the series key more than once.  L1831-1832 are the largest contributors to allocations in this part of the code.